components/pixels/lib.rs

```diff
@@ -283,7 +284,8 @@ pub struct RasterImage {
     pub format: PixelFormat,
     pub id: Option<ImageKey>,
     pub cors_status: CorsStatus,
-    pub bytes: IpcSharedMemory,
+    #[conditional_malloc_size_of]
+    pub bytes: Arc<IpcSharedMemory>,
```

Comment:
Hm, isn't IpcSharedMemory internally Arced (or at least like it acts like Rc, so it has cheap clone that still points to same backing).